spec/index.html

```diff
@@ -758,7 +758,7 @@ <h3>Triples</h3>
 
     <p>The three components (|s|, |p|, |o|) of an [=RDF triple=] are respectively called the <dfn class=export >subject</dfn>, <dfn class=export >predicate</dfn> and <dfn class=export >object</dfn> of the triple.</p>
 
-    <p>The set of <span id="dfn-nodes"><!-- obsolete term--></span><dfn class="export" data-lt="RDF node">nodes</dfn> of an <a>RDF graph</a>
+    <p>The set of <span id="dfn-nodes"><!-- obsolete term--></span><dfn id="dfn-node" class="export" data-lt="RDF node">nodes</dfn> of an <a>RDF graph</a>
```

Comment:
```suggestion
    <p>The set of <span id="dfn-rdf-node"></span> <span id="dfn-nodes"></span><dfn id="dfn-node" class="export" data-lt="RDF node">nodes</dfn> of an <a>RDF graph</a>
```
to match #215.

There may be a better way to write this in reSpec.

Reply:
This looks like it sets two anchors, both “dfn-node”. I think the span should be dfn-node, but the dfn be dfn-rdf-node. When searching for a term, at least theoretically, it’s the dfn version (when exported). Anything else remains for historic references.

Reply:
The suggestion change is adding`dfn-rdf-node` that is, for a few days only, used in SHACL 1.2 Core.

3 different anchors : `dfn-nodes`, `dfn-rdf-node`, `dfn-node`.

RDF 1.1 : `#dfn-node`
RDF 1.2 current, post 215 : `#dfn-nodes` 
RDF 1.2 current, post 215 : `#dfn-rdf-node``

We could make a breaking change of new items introduced recently as long as `#dfn-nodes` works for RDF 1.1 TR/rdf-concepts.

Reply:
I was working under the assumption that the anchors that existed only for a few days after #215 was merged could safely be dropped, but the SHACL example proved me wrong, so I merged Andy's proposal to be on the safe side...

@afs do you think that we also need to re-introduced `#dfn-full-conformance` and `#dfn-basic-conformance`?

Reply:
SHACL core has updated to `dfn-rdf-literal`, `dfn-rdf-node` (singular). https://github.com/w3c/data-shapes/pull/437

> about dfn-node or dfn-literal

`dfn-node` (singular) and `dfn-literal` are in RDF 1.1 Concepts.

So we can keep exactly those as `<span>`, and remove the `dfn-nodes` before it leaks.

> @afs do you think that we also need to re-introduced `#dfn-full-conformance` and `#dfn-basic-conformance`?

I don't know of any reason to. These are new concepts.

I'm sure exactly how data-lt works so writingit out, is this what we want?

```<span id="dfn-rdf-node"></span><dfn id="dfn-rdf-node" class="export" data-lt="RDF node">nodes</dfn>```
  
```<span id="dfn-literal"></span><dfn id="dfn-rdf-literal" class="export" data-lt="RDF literal">literal</dfn>```

Reply:
I think it's important to keep the `<!-- obsolete term-->` in the otherwise empty `<span>`, to remind future editors (including ourselves) why that empty `<span>` is there, such that it not be removed without otherwise addressing the reason for its presence.

I *think* I've absorbed how these work well enough to provide the following guidance. If I'm in error anywhere, please forgive and correct me.

The empty `<span>` gets the `@id` previously based on the old term — e.g., `id="dfn-literal"` — and the full `<dfn>` that follows gets an `@id` based on the new term — e.g., `id="dfn-rdf-literal"`. This keeps `@href`s from existing documents that target the old `@id` from breaking.

The `@data-lt` is required when the visible text (e.g., `nodes`) doesn't exactly match the term we mean to define (e.g., `data-lt="RDF node"`) occurrences of which elsewhere can be marked to link to this definition with plain wrappers of `<a> </a>`.

The `@data-lt` value *can* include multiple strings (separated by `|`) so multiple "source" texts (which need not include the visible text, nor most regular plurals/singular variants, because ReSpec already handles these; irregular plurals/singular variants are sometimes necessary) can be linked to here.